Volume/Views/MainView/PublicationList.swift

```diff
@@ -27,24 +25,32 @@ struct PublicationList: View {
         }
     }
     
-    // The publications a user is not following
+    /// The publications a user is not following
     private var notFollowedPublications: some View {
-        Section {
-            Header(text: "More Publications")
-            ForEach(publicationsData) { publication in
-                NavigationLink(destination: PublicationDetail(publication: publication)) {
-                    MorePublicationRow(publication: publication)
-                        .buttonStyle(PlainButtonStyle())
-                        .padding(.trailing)
+        Section(header: Header("More publications")) {
+            LazyVStack {
+                ForEach(0..<15) { i in
+                    let publication = publicationsData[i % publicationsData.count]
+                    NavigationLink(destination: PublicationDetail(publication: publication)) {
+                        MorePublicationRow(publication: publication, onToggleFollowing: onToggleFollowing)
+                            .padding(.bottom, 15)
+                    }
                 }
             }
         }
     }
     
+    /// Toggle following this Publication
+    private func onToggleFollowing(publication: Publication) {
+        
+    }
+    
     var body: some View {
         NavigationView {
             ScrollView(showsIndicators: false) {
                 followedPublications
+                Spacer()
+                    .frame(height: 16)
                 notFollowedPublications
```

Comment:
`Spacer(minlength: 16)` as a one-liner?

Reply:
I think `minLength` would specify the minimum height of the spacer, but I want it to be exactly that height.